ktor-client/ktor-client-features/ktor-client-auth/build.gradle.kts

```diff
@@ -1,23 +1,8 @@
+/*
+ * Copyright 2014-2020 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
 description = "Ktor client Auth support"
 
 kotlin.sourceSets {
-    commonMain {
```

Comment:
why we removed the dependencies?

Reply:
Becaude we add them in client-features for all submodules